android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/splash/SplashScreen.kt

```diff
@@ -0,0 +1,44 @@
+package co.nimblehq.ic.kmm.suv.android.ui.screens.splash
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.ic.kmm.suv.android.R
+import kotlinx.coroutines.delay
+
+@Composable
+fun SplashScreen(onDidLaunch: () -> Unit) {
```

Comment:
`onDidLaunch` 🙄 This shows how iOS devs apply their style in the different language 🤣

Reply:
Lol, changed it to `onLaunch`

Fixed in https://github.com/suho/kmm-ic/pull/79/commits/30bef640c4329f8860ab9d30b9e6134791383f58